### PR TITLE
WIP extra steps json

### PR DESCRIPTION
**Pull request by blaggacao**

### CODE REVIEW

File: src/bitte_ci/runner.cr [hunk 1]
```diff
@@ -146,6 +156,47 @@ module BitteCI
...
+        then {
+          ${cmd.name} = {
+            command = [ "#{entrypoint}" "--pure" cmd.name];
+            label = cmd.help;
+          };
+        }
+        else {}
+      );
+      devshellSteps = builtins.foldl' op {}
+      NIX
```
Comment (blaggacao) [inline src/bitte_ci/runner.cr:184]: I think the flake should provide the info in the right shape instead...

File: src/bitte_ci/runner.cr [hunk 2]
```diff
@@ -146,6 +156,47 @@ module BitteCI
...
 
+    def fetch_devshell_steps_json
+      full_name = @pr.base.repo.full_name
+      sha = @pr.base.sha
+
+      
+
+      entrypoint_mem = IO::Memory.new
+      status = Process.run("nix", output: entrypoint_mem, error: STDERR,
+        args: ["eval", "--raw", "github:#{full_name}/#{sha}.#devShell.x86_64-linux.outPath"])
```
Comment (blaggacao) [inline src/bitte_ci/runner.cr:167]: How do we get the correct `system` from this environment?

Comment (nrdxp) [reply src/bitte_ci/runner.cr:167]: would a call to `nix eval --impure --raw --expr "builtins.currentSystem"` work?

Comment (blaggacao) [reply src/bitte_ci/runner.cr:167]: Sounds good! Thx

**Comment by blaggacao** [comment]
Abandoning this, because I swapped a strategic significance of `numtide/devshell` of 10 for a strategic significance of `fluidattacks/makes` of 100.

This was made with `numtide/devshell` in mind

**Pull request closed without merging**